src/components/Dropdown/Dropdown.tsx

```diff
@@ -16,8 +16,7 @@ export interface IDropdownState {
 export class Dropdown extends React.Component<IDropdownProps, any> {
```

Comment:
Should all of the components be subclassed from BaseComponent? I see that's where we're adding `dir='rtl'` to the document, which is required by all/most of the components.